web/client/epics/identify.js

```diff
@@ -386,7 +386,7 @@ export const mouseMoveMapEventEpic = (action$, {getState}) =>
         .switchMap(({position, layer}) => {
             const isAnnotationsEnabled = createControlEnabledSelector('annotations')(getState());
             const isMeasureEnabled = measureSelector(getState());
-            const isMouseOut = getState().mousePosition.mouseOut;
+            const isMouseOut = getState()?.mousePosition?.mouseOut;
```

Comment:
This should be externalized in a selector. IMHO

Reply:
ah the eternal debate on DRY for selectors ;)

IMO this can be externalized, but is not really important since it is used only here

Reply:
Not about DRY (used only once) but about encapsulation

Reply:
ok please use selectors inside this epic, ignore the rest of the file
and keep this as a good practice